press/press/doctype/payout_order/payout_order.py

```diff
@@ -57,87 +59,126 @@ class PayoutOrder(Document):
 	@staticmethod
 	def get_list_query(query):
 		PayoutOrder = frappe.qb.DocType("Payout Order")
-		query = query.where((PayoutOrder.docstatus != 2))
-		return query
+		return query.where(PayoutOrder.docstatus != 2)
 
 	def validate(self):
-		self.validate_items()
-		self.validate_net_totals()
-		self.compute_total_amount()
-
-	def validate_items(self):
 		for row in self.items:
```

Comment:
this shouldn't be under validate but rather some `calculate_net_total`

Nothing is being validated here